networkx/readwrite/tests/test_gexf.py

```diff
@@ -400,11 +400,11 @@ def test_edge_id_construct(self):
     def test_numpy_type(self):
         G = nx.path_graph(4)
         try:
-            import numpy
+            import numpy as np
         except ImportError:
             return
```

Comment:
+1 on the numpy alias, but I think the whole try/except block could be replaced with `pytest.importorskip`, e.g.
```python
def test_numpy_type(self):
    np = pytest.importorskip("numpy")
    G = nx.path_graph(4)
    ...
```